build/gulp/tasks/docs.ts

```diff
@@ -78,7 +91,12 @@ task('build:docs:example-menu', () =>
 
 task(
   'build:docs:json',
-  parallel('build:docs:docgen', 'build:docs:component-menu', 'build:docs:example-menu'),
+  parallel(
```

Comment:
Watch (`yarn start`) does not reflect changes if Behavior description changes.

Reply:
good point :)
I was trying to implement that watch will rebuild either behaviors file, but was not able to find how to do it.
@miroslavstastny or @kuzhelov pleae if you could help me with it?

Reply:
I added watch for behaviours. Watch currently does not work on Windows (not caused by this PR), I will address that in #172 once this PR is merged.